client/src/components/grants/Details.tsx

```diff
@@ -59,88 +100,110 @@ export default function Details({
+                src="./assets/twitter_logo.svg"
+                alt="Twitter Logo"
+              />
+              <a
+                className="mr-2 text-primary-background"
+                target="_blank"
+                href={`https://twitter.com/${project?.projectTwitter}`}
+                rel="noreferrer"
+              >
+                {project?.projectTwitter}
+              </a>
+              {project?.credentials?.twitter && <Verified />}
+            </div>
+          )}
+          {project?.projectGithub && (
+            <div className="flex justify-start items-center m-2">
+              <img
+                className="h-4 mr-2"
+                src="./assets/github_logo.png"
+                alt="Github Logo"
+              />
+              <a
+                className="mr-2 text-primary-background"
+                target="_blank"
+                href={`https://github.com/${project?.projectGithub}`}
+                rel="noreferrer"
+              >
+                {project?.projectGithub}
+              </a>
+              {project?.credentials?.github && <Verified />}
+            </div>
+          )}
+        </Box>
+        <Box>
+          <div>
+            <p className="flex text-sm m-2 pb-2">
+              <Calendar color={colors["secondary-text"]} />
+              <span className="ml-1">Created on: {createdAt}</span>
+            </p>
           </div>
-        )}
-        {project?.projectGithub && (
-          <div className="flex justify-start items-center">
-            <img
-              className="h-4 mr-2 mt-1"
-              src="./assets/github_logo.png"
-              alt="Github Logo"
-            />
-            <a
-              className="mr-2 text-primary-background"
-              target="_blank"
-              href={`https://github.com/${project?.userGithub}`}
-              rel="noreferrer"
-            >
-              {project?.userGithub}
-            </a>
+          <div>
+            <p className="flex text-sm m-2 pb-1">
+              <Calendar color={colors["secondary-text"]} />
+              <span className="ml-1">Last Edited: {updatedAt}</span>
+            </p>
           </div>
-        )}
-      </div>
-
+          {project?.projectGithub && (
+            <div className="flex items-center m-2">
+              <img
+                className="h-4 ml-0.5 mr-2 mt-1"
+                src="./assets/github_logo.png"
+                alt="Github Logo"
+              />
+              <a
+                className="text-primary-background"
+                target="_blank"
+                href={`https://github.com/${project?.userGithub}`}
+                rel="noreferrer"
+              >
+                {project?.userGithub}
+              </a>
+            </div>
+          )}
+        </Box>
+        <Box>
+          <Box p={1}>
+            <span className="text-[20px]">My Applications</span>
+          </Box>
+          <Box>
+            {props.applications.length > 0 &&
```

Comment:
Won't this always be > 0? The initial state contains an empty one and I think that's what is causing to show an unknown application?

<img width="460" alt="Schermata 2022-10-26 alle 08 48 44" src="https://user-images.githubusercontent.com/129965/198073891-f8754b2d-4d6c-4300-a807-985c729b95b2.png">

Reply:
I updated the initial state value to []